ui/app/store/actions.js

```diff
@@ -2822,6 +2822,10 @@ export function getCurrentWindowTab() {
   };
 }
 
+export function setLedgerLivePreference(value) {
+  return async () => promisifiedBackground.setLedgerLivePreference(value);
```

Comment:
We really need to create a distinction by filename or something when dealing with actions like this that do not require redux or a thunk. In many PRs from many authors (myself included) this pattern has emerged of dispatching an action that only must be dispatched because it is constructed as a thunk like this.

Perhaps we need like a `background-actions.js` file for interacting with the promisifiedBackground? cc @Gudahtt

formal [REQ]: This doesn't need to be a traditional redux action. Return the call to `promisifed.setLedgerLivePreference` directly without the intermediary function. This will also mean that this action doesn't need to be included in `mapDispatchToProps` and can be imported and used directly.